app/models/cloud_subnet_network_port.rb

```diff
@@ -0,0 +1,6 @@
+class CloudSubnetNetworkPort < ApplicationRecord
+  self.table_name = "cloud_subnets_network_ports"
+
+  belongs_to :cloud_subnet
+  belongs_to :network_port
+end
```

Comment:
the CloudSubnetNetworkPort holds an IP address. So I think it needs to be this. Might be that more relations will be tied to this. E.g. floating IP is associated to port and fixed ip, so might be this model is better for the association